.golangci.yml

```diff
@@ -2,54 +2,67 @@ run:
   tests: true
   #   # timeout for analysis, e.g. 30s, 5m, default is 1m
   timeout: 5m
+  allow-parallel-runners: true
 
 linters:
   disable-all: true
   enable:
+    - asciicheck
```

Comment:
do we need the rest of these linters? Ideally, since `gci` does seem like a good addition, it would be good to separate from the rest, they could be decided in a separate PR.

Reply:
great idea @DimitrisJim